### PR TITLE
Add blog post about contribution sprints for 2024.

### PR DESCRIPTION
**Pull request by tim-schilling**

### CODE REVIEW

**Comment by keanyaphelps** [comment]
@tim-schilling  LGTM!

File: src/_content/posts/contribution-sprints.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+author: DjangoCon US Communications Team
+category: General
+published_datetime: 2024-09-010 12:00:00-04:00
```
Comment (drewbrew) [inline src/_content/posts/contribution-sprints.md]:
```suggestion
published_datetime: 2024-09-10 12:00:00-04:00
```

File: src/_content/posts/contribution-sprints.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+layout: post
+title: Contribute to Django at the Contribution Sprints!
+
+cover:
+    url: /assets/img/blog/djangocon_us_sprints_closeup_2022.jpg
+    alt: "Two Development Sprints attendees at DjangoCon US 2022 in San Diego, chatting."
+---
+
+We're excited to talk to you about contribution sprints at DjangoCon US! They are being held Thursday, September 26th and Friday, September 27th at the Caktus offices and virtually on our conference Slack. They are free to attend and open to everyone.
```
Comment (drewbrew) [inline src/_content/posts/contribution-sprints.md]:
```suggestion
We're excited to talk to you about contribution sprints at DjangoCon US! They are being held Thursday, September 26th and Friday, September 27th at the [Caktus Group offices](https://maps.app.goo.gl/wwwXU12mFpNCph5n6) and virtually on our conference Slack. They are free to attend and open to everyone.
```

Adding a Google Maps walking directions link

**Review by drewbrew — APPROVED** [review]
couple minor tweaks

**Review by jcjudkins — APPROVED** [review]
Looks good!

**Pull request merged**